cmd-car-split.go

```diff
@@ -324,3 +358,22 @@ func writeNode(node datamodel.Node, w io.Writer) (cid.Cid, error) {
 	}
 	return cd, nil
 }
+
+func writeMetadata(metadata Metadata, epoch int) error {
```

Comment:
I think the metadata should be written just once, after all is done.

The encoding should be as array of `Metadata`.

Reply:
Fixed